src/main/java/com/example/cartservice/dto/CreateCartDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.example.cartservice.dto;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public class CreateCartDTO {
```

Comment:
According to class naming rules you should use nouns for class names. In that case not **CreateCartDTO** but you should use **CartDTO**

Reply:
Do not create separate DTOs for each different request. You can use one DTO for creating, deleting and updating. Use just CartDTO

Reply:
oh I see